jobs/frontend/senior-frontend-developer.md

```diff
@@ -14,17 +14,38 @@ We're looking for a pro-active senior-level frontend developer, who not only sol
 ## We expect that you
 
 - Share our values ([en](https://datarockets.com/blog/company/datarockets-codex-core-values/), [ru](https://docs.google.com/document/d/1DojRoGnl-J0Ku0B9tjK_-HUMbKQDPoueoYr645dYA-4/edit)) – it's very important.
+- Write readable and maintainable code.
+- Care about user experience.
+- Have knowledge of best practices in development and in UI/UX design.
 - Can design a whole system of application.
-- Knows at least one library/framework for building UI at advanced level (React, Vue, Angular, etc.).
-- Have experience working with different kind of teams and projects.
-- Research new technologies and promote them inside the team (new libraries, tools, techniques: refactoring, debugging, etc.).
-- Will constantly suggest innovations on the project.
-- It will be a plus if you released own projects.
+- Have proficient knowledge in HTML/CSS/JS.
+- Know at least one library/framework for building UI at advanced level (React, Vue, Angular, etc.).
+- Know how browsers work.
+- Have attention to details.
+- Can learn new technologies as you go.
+- Research new technologies (new libraries, tools, techniques: refactoring, debugging, etc.).
 - Understand yourself: have good habits, know how to be productive and how to rest.
 - Have knowledge of English on the level which allows you to feel comfortable communicating in it (verbally and written) the whole working day.
 - Know how important it is to communicate with the team and inform others about what you are doing, so you stick to open and frequent communication. It is important for us.
 
-It will be a plus if you keen to become a Team Lead or already have this experience and know what skills are necessary to be a good Team Lead.
+## It will be a plus if you
+
+- Are keen to become a Team Lead or already have this experience and know what skills are necessary to be a good Team Lead.
+- Released own projects.
+- Share knowledge with others by writing blog posts, participating in conferences, etc.
+- Have experience in Backend.
+- Have experience in UI/UX design.
+- Have experience in profiling performance and optimizing it.
+
+## Technologies we use
+
+The tech stack highly depends upon a project. But there are some technologies that we usually prefer for Frontend:
+
+- For building UI, we use `React` with `Next.js`.
+- For styling, we use either `CSS-in-JS` or `CSS Modules`.
+- For type checking, we use `TypeScript`.
+- For DX, we use `Prettier`, `ESLint`, `Storybook`, configure CI/CD via `Github Actions`.
+- For testing, we usually use `Jest`, `Testing Library`, `Cypress`/`Playwright`.
```

Comment:
@ezhlobo Can you please review this section? I'm not sure that this section looks good.

Reply:
@lesha1201 – I'm not Eugene but from what I know about other projects, this is very true information. Just my opinion, you can merge the PR at this stage just easily so that we can proceed, and then this list can be improved asynchronously in further PRs.

Reply:
@atatarenko Ok. Let's merge it then. I can't merge it because I don't have access to `datarockets/career`. Can you please merge it?